fix broken test by tweaking floating point value that is slightly off
see also https://github.com/openbabel/openbabel/issues/2246
author: Kenneth Hoste (HPC-UGent)
--- openbabel-openbabel-3-1-1/test/testobconv_writers.py.orig	2022-01-12 20:12:01.480069896 +0100
+++ openbabel-openbabel-3-1-1/test/testobconv_writers.py	2022-01-12 20:12:12.180174159 +0100
@@ -2876,7 +2876,7 @@
                 2.4146659588503769,
                 4.189331679349326,
                 4.052466878708012,
-                -1.4648575597102012
+                -1.4648575597102014
               ],
               "y": [
                 1.5846,
@@ -2891,7 +2891,7 @@
                 2.4146659588503769,
                 4.189331679349326,
                 4.052466878708012,
-                -1.4648575597102012
+                -1.4648575597102014
               ],
               "style": {
                 "annotation": [
